tests/v1/math/test_math.py

```diff
@@ -24,6 +24,21 @@ def test_parse_simple():
     assert float(sympify_petab("1 + 2 * (3 + 4) / 2")) == 8
 
 
+def test_assumptions():
+    # in PEtab, all symbols are expected to be real-valued
+    assert sympify_petab("x").is_real
+
+    # non-real symbols are changed to real
+    # TODO: should we raise an error instead?
+    assert sympify_petab(sp.Symbol("x", real=False)).is_real
```

Comment:
Probably raise an error, or does it occur in standard use?

Reply:
It happens if you pass `sympify("x")`. Not sure what standard use will be :sweat_smile:

Reply:
Ah, fine for me on the level of symbols. I was thinking of expressions, which I guess is irrelevant here, so nevermind.